index.js

```diff
@@ -54,11 +54,14 @@ class KeepAChangelog extends Plugin {
   }
 
   beforeRelease() {
-    const { version } = this.getContext();
-    const formattedDate = getFormattedDate();
-    const releaseTitle = `\n\n## [${version}] - ${formattedDate}\n\n`;
-    const changelog = this.changelogContent.replace(this.unreleasedTitle, releaseTitle);
-    fs.writeFileSync(this.changelogPath, changelog);
+    const { isDryRun } = this.global;
+    if (!isDryRun) {
```

Comment:
Could you please do `if(isDryRun) return;` I do this everywhere in my projects. Sorry for nitpicking.

Reply:
Hey no worries! Now fixed.